packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2310,6 +2311,15 @@ export function enqueuePendingPassiveHookEffectUnmount(
 ): void {
   if (runAllPassiveEffectDestroysBeforeCreates) {
     pendingPassiveHookEffectsUnmount.push(effect, fiber);
+    if (__DEV__) {
+      if (deferPassiveEffectCleanupDuringUnmount) {
+        fiber.effectTag |= PassiveUnmountPendingDev;
+        const alternate = fiber.alternate;
```

Comment:
Nit: Can you pass `current` as an argument through `schedulePassiveEffects`? We try to limit the number of places we access the `.alternate` pointer, since one day we're hoping to remove i t.

Reply:
Ah, sure, although this is going to require adding a second param to a few other functions (e.g. `commitUnmount`, `commitNestedUnmounts`) since those also call `enqueuePendingPassiveHookEffectUnmount`.

Reply:
I could do that for `schedulePassiveEffects()` because `commitLifeCycles()` already has both fibers.

But we also call `enqueuePendingPassiveHookEffectUnmount()` from `commitUnmount()` which is called from `commitNestedUnmounts()` which traces all the through to `commitMutationEffects()`, so I'm still going to have to access `alternate` there (and add a bunch of additional params to the functions in between). 

Is this still the right change?

Reply:
It looks like what we're calling "current" is inconsistent between `commitWork` and `commitDeletion` calls.

For e.g. `commitWork` we pass `nextEffect.alternate` as `current`:
https://github.com/facebook/react/blob/cfefc81ab2f5103bedc9e45701e8c00c0689f499/packages/react-reconciler/src/ReactFiberWorkLoop.new.js#L2192-L2193

But for `commitDeletion` we pass `nextEffect` as `current`:
https://github.com/facebook/react/blob/cfefc81ab2f5103bedc9e45701e8c00c0689f499/packages/react-reconciler/src/ReactFiberWorkLoop.new.js#L2214

Reply:
We also call `commitNestedUnmounts` and `commitUnmount` from `unmountHostComponents` while traversing the tree, so we'd need to add a bunch of additional `alternate` access here to pass through that prop.

I think I'm going to *not* make this change for now. Seem reasonable?

Reply:
Right I guess “current” isn’t accurate, it’s just the “other fiber”

Yeah that makes sense. If we got rid of alternates, each fiber would have a shared instance object and we could put the flag on that.